action_tutorials/action_tutorials_py/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.5)
+project(action_tutorials_py)
```

Comment:
Since this package only contains Python code, we don't need it to be a CMake package.
I think all we need to do is add a `setup.py` (see [demo_nodes_py](https://github.com/ros2/demos/tree/master/demo_nodes_py) for example) and change the build type in the `package.xml`.